### ISSUE TITLE
Enhancement Chip roms are still imported even with the --only-retail flag.

### ISSUE THREAD

**Issue by baglio** [emmercm/igir#1342]
### Paste the command

```zsh
igir copy \
-i "snesroms" \
-o "output" \
--dat "No-Intro Love Pack (PC XML) (2024-09-12).zip" \
-R "USA,ITA,WORLD" \
--only-retail --no-bios --no-device --no-unlicensed \
--dir-dat-name
```

### Describe the bug

Games that have `(Enhancement Chip)` label are copied in the output folder.

### Expected behavior

Games that have `(Enhancement Chip)` label, which are not retail games, should not be copied.

### Debug logs

[igir-single-dat.log](https://github.com/user-attachments/files/16978400/igir-single-dat.log)

### DAT(s) used

No-Intro Love Pack (PC XML) (2024-09-12).zip

### igir version

3.0.0

### Node.js version

18.18

### Operating system

macOS 14.6

### Additional context

Igir installed via Homebrew.

**Comment by emmercm** [comment]
Interesting, I didn't realize those SNES chips had ROMs and were cataloged.

Question for you: would you consider those BIOS files? I would consider them "retail" as in they're official, so I'm hesitant to exclude them that way.

**Comment by baglio** [comment]
Good question.
My knowledge is limited in this matter, but since we are talking about hardware and not software I'm not sure if those Enhancement chips belong to the BIOS category.

I personally consider “retail” anything that a store or merchant could deliver into the hands of a regular customer.

For the same reason I would also exclude any (Switch Version), (Virtual Console) etc, but maybe that's too far, is just a preference of mine.

**Comment by emmercm** [comment]
Let me think on this.

I probably won't consider these files "BIOS" as they wouldn't be necessary for an emulator, i.e. I wouldn't copy them into RetroArch's "system" directory.

I see the argument for those re-releases. "Retail" was the best word I could come up with to indicate non-prototype, non-demo, non-aftermarket, non-cracked, non-translated, etc, etc. Language is hard 🙂.

**Comment by baglio** [comment]
> I see the argument for those re-releases. "Retail" was the best word I could come up with to indicate non-prototype, non-demo, non-aftermarket, non-cracked, non-translated, etc, etc. Language is hard 🙂.

Yes, honestly "retail" is the best word for describing non-prototype, non-demo, non-aftermarket, non-cracked, non-translated, etc. The only thing I can come up with is the word "Commercial" to describe everything that could have been sold by a merchant or store at that time.
